src/main/java/nextstep/courses/domain/SessionImage/ImageCapacity.java

```diff
@@ -0,0 +1,19 @@
+package nextstep.courses.domain.SessionImage;
+
+public class ImageCapacity {
+
+    public static final int MINIMUM_SIZE = 0;
+    public static final int MAXIMUM_SIZE = 1;
+    private final int imageSize;
```

Comment:
```suggestion
public class ImageCapacity {

    public static final int MINIMUM_SIZE = 0;
    public static final int MAXIMUM_SIZE = 1;
    
    private final int imageSize;
```

상수와 인스턴스 변수 간 개행으로 구분해 보면 어떨까요?

Reply:
현재는 MB가 기본단위이므로 사진 사이즈가 50kb라면 처리가 불가능할 것 같아요

보통 파일 사이즈는 최소 단위인 Byte로 처리되는데 기본 단위를 바꿔보면 어떨까싶어요

Reply:
둘다 반영완료했습니다.
파일의 보편적 사이즈인 byte로 단위를 바꿔보았습니다.